### PR TITLE
Fix: avax shows in wrong position in list

### PR DESCRIPTION
**Pull request by prodesert22**
If exist avax in search in watchlist widget, move avax to the top of list

### CODE REVIEW

**Comment by HariSeldon23** [comment]
Task linked: [CU-256trn7 Watchlist minor fix](https://app.clickup.com/t/256trn7)

File: src/pages/Beta/Swap/WatchList/CurrencyPopover/index.tsx [hunk 1]
```diff
@@ -70,6 +70,15 @@ const CurrencyPopover: React.FC<Props> = ({
     if (searchToken) return [searchToken]
     const sorted = filteredTokens.sort(tokenComparator)
 
+    // if avax in sorted tokens, move it to the top
+    const containAvax = sorted.some(token => token.symbol === 'WAVAX')
+    if (containAvax) {
+      const avaxIndex = sorted.findIndex(token => token.symbol === 'WAVAX')
```
Comment (bmino) [inline src/pages/Beta/Swap/WatchList/CurrencyPopover/index.tsx:76]: shouldn't this re-sort be addressed as a parm in `useTokenComparator`?

if we are indeed modifying the data here, we can avoid iterating `sorted` twice by something like:
```javascript
const avaxIndex = sorted.findIndex(token => token.symbol === 'WAVAX')
if (avaxIndex > 0) { // Index of 0 means it is already first
    const avax = filteredTokens[avaxIndex]
    sorted.splice(avaxIndex, 1)
    sorted.unshift(avax)
}
```

File: src/pages/Beta/Swap/WatchList/CurrencyPopover/index.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ const CurrencyPopover: React.FC<Props> = ({
   const isAddressSearch = isAddress(searchQuery)
   const searchToken = useToken(searchQuery)
 
-  const tokenComparator = useTokenComparator(invertSearchOrder)
+  const tokenComparator = useTokenComparator(invertSearchOrder, WAVAX[useChainId()])
```
Comment (bmino) [inline src/pages/Beta/Swap/WatchList/CurrencyPopover/index.tsx:62]: is this valid React?

Comment (prodesert22) [reply src/pages/Beta/Swap/WatchList/CurrencyPopover/index.tsx:62]: It's valid but it's ugly 🤣

**Comment by HariSeldon23** [comment]
@StarlordPangolin please comment if you need any more information to add this to our test cases

**Comment by StarlordPangolin** [comment]
> @StarlordPangolin please comment if you need any more information to add this to our test cases

I got this and have added it to the test cases.

**Pull request merged**